### DISCUSSION TITLE
Does the July 4th update change the previous watch face?

### DISCUSSION THREAD

**Discussion by gearmesh** [evs-chris/Hearty-McWeatherface#6] [General]
Does the July 4th update change the previous watch face? I see a lot more code in the index.view file.

I took your original code but was enlarging the font for the 5 items near the bottom, steps. stairs etc.
I still haven't figured out how to move the text of the counts more right of their icons.
![Fitbit lg font](https://user-images.githubusercontent.com/5623590/132155125-a85c270e-3756-4c31-bcea-1c4b26d30a72.jpg)

Greg

**Comment by evs-chris** [comment]
I don't think it did a whole lot to the main face, but there have been a few additional views added. There's a change log in docs that breaks everything down. I've started a set of changes to break the view into multiple files, but it's a pretty pervasive change and I haven't had much time to work on it.
